tests/acceptance/fixtures.py

```diff
@@ -579,23 +579,30 @@ def test_foo():
 
 
 @pytest.fixture(autouse=True)
-def only_with_distro_feature(request, bitbake_variables):
-    """Fixture that enables use of `only_with_distro_feature(feature1, feature2)` mark.
+def only_with_mender_feature(request, bitbake_variables):
+    """Fixture that enables use of `only_with_mender_feature(feature1, feature2)` mark.
     Example::
 
-       @pytest.mark.only_with_distro_feature('mender-uboot')
+       @pytest.mark.only_with_mender_feature('mender-uboot')
        def test_foo():
            # executes only if mender-uboot feature is enabled
            pass
 
     """
 
-    mark = request.node.get_closest_marker("only_with_distro_feature")
+    mark = request.node.get_closest_marker("only_with_mender_feature")
```

Comment:
This won't work for tests using `only_with_distro_feature` marker, will it?

Reply:
Of course, this was a totally lame attempt, I must be tired! It should be fixed now!